packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -622,6 +696,33 @@ class VectorGraphicsCodec {
     final double height = buffer.getFloat32();
     listener?.onSize(width, height);
   }
+
+  void _readTextConfig(
+    _ReadBuffer buffer,
+    VectorGraphicsCodecListener? listener,
+  ) {
+    final int id = buffer.getUint16();
+    final double dx = buffer.getFloat32();
+    final double dy = buffer.getFloat32();
```

Comment:
nit: I think at some point we'll want to support relative positioning, so we probably want to avoid calling this `dx` to avoid confusion. Same for dy.